lib/msal-angular/src/msal.service.ts

```diff
@@ -24,7 +24,7 @@ export class MsalService extends UserAgentApplication {
     private loginScopes: string[];
     _renewActive: boolean;
 
-    constructor(@Inject(MSAL_CONFIG) private config: MsalConfig, private router: Router, private broadcastService: BroadcastService) {
+    constructor(@Inject(MSAL_CONFIG) private config: MsalConfig, private broadcastService: BroadcastService) {
```

Comment:
Is this a breaking change (changing the constructor function signature)? Not familiar enough with Angular to know.

Reply:
It is if users are instantiating the msal service themselves! For the most part, the injector will take care of instantiating the service and users dont have to know about this change.

Reply:
Gotya. Given `MsalService` is exported and thus part of the public API, I'd like to avoid a breaking change, even if we don't think anyone is actually using the API in that way. 

If `router` was left in the function signature (to keep the same function signature), presumably if an app was instantiating `MsalService` manually, that `router` would no longer be used, correct?

Reply:
I agree with @jasonnutter to keep the surface intact as the release (once this PR goes through approval) wouldn't be a major version release.

Reply:
changed the constructor definition back to include router, defaulted it to null..
Can you please review it soon so that we can make our build pick it up from the public registry?

Reply:
Hi @jasonnutter can you please let me know if there is anything pending on this PR from our side?

Reply:
I'll have the team take a look at this PR and get it approved, apologies for the delay!